ui/src/literature/components/LiteratureCollectionBanner.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert } from 'antd';
+import ExternalLink from '../../common/components/ExternalLink.tsx';
+
+function HiddenCollectionAlert({ collections }) {
+  const shouldDisplayAlert = collections !== 'Literature';
```

Comment:
```collections``` is an array 😅 for example
```json
{
"_collections": [
    "Literature",
    "Fermilab"
  ],
}

```

Looking at the code more carefully, we don't make `_collections` [available](https://github.com/inspirehep/inspirehep/blob/140f7323cc0f12c51b334912743bfbb39cb2cb33/backend/inspirehep/records/marshmallow/literature/base.py#L42) to the ui serializer, which means `_collections` is not in the response. What we can do is the following, since we don't want to make this information available to users without permission, we could add a new boolean field in the [ui serializer](https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/records/marshmallow/literature/ui.py#L95) called `is_collection_hidden`:


```python

 is_collection_hidden = fields.Method("get_ is_collection_hidden")

def is_collection_hidden(self, data, original_data):
    return True or False
```

Then in the UI we could read this and display or not the banner.